recipe/meta.yaml

```diff
@@ -11,32 +11,33 @@ source:
   sha256: 5fe59118c7fe5373ae08a2360abb6f2a6891b6eb4235b76b3461f0427b5b29b0
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - pip
+    - python >=3.8,<4.0
     - poetry >=1.0.2
-    - python >=3.8
+    - poetry-core
+    - pip
   run:
-    - overrides >=3.0.0,<4.0.0
-    - python >=3.8
+    - python >=3.8.0,<4.0.0
     - requests <3.0.0
   run_constrained:
-    - janome >=0.3.10,<0.4.0
+    - janome >=0.5.0,<0.6.0
     - natto-py >=1.0.0,<2.0.0
     - kytea >=0.1.4,<0.2.0
     - sentencepiece >=0.1.85,<0.2.0
-    - sudachipy ==0.4.9
-    - boto3 >=1.11.0,<2.0.0
-    - fastapi >=0.54.1,<0.66.0
-    - uvicorn >=0.13.4,<0.14.0
-    - sudachidict-core >=20200330,<20200331
-    - sphinx >=3.1.1,<3.2.0
-    - sphinx_rtd_theme >=0.4.3,<0.5.0
+    - sudachipy >=0.5.0,<0.6.0
+    - boto3 >=1.34.0,<2.0.0
+    - fastapi <1.0.0
+    - uvicorn <0.26.0
+    - sudachidict-core 20230927
+    - sphinx >=5.1.1,<6.0.0
+    - pydata-sphinx-theme >=0.10.1,<0.11.0
     - nagisa >=0.2.7,<0.3.0
+    - rich >=13.7.0,<14.0.0
```

Comment:
Generated using grayskull (`grayskull pypi konoha`).

Reply:
Looks great, thank you. Did you also manually verify if it is correct? Grayskul is not always 100%.

Reply:
Sorry for the 🔰 question but how should we verify?

At least, I manually checked `pyproject.toml` on upstream and `meta.yaml` and these were aligned after 47d4284. 📝

Reply:
Oh yeah that’s all I meant, just a quick double check. 😂